Packages/ANCIENT-MEGALITHIC-MYSTERY-PACKAGE/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="atg-location" content="Cusco">
```

Comment:
https://www.ticketmachupicchu.com/cusco-cuzco/

Reply:
atg-location - hmmm... https://gist.github.com/lancejpollard/1978404

Reply:
where is the shortcut icon?

Reply:
1. ok ill change with z
2. That meta tag is for fetching the package by location, right now is only mock data so i have to fake API data.
3. shortcut icon? is that the favicon?